deep-learning/_papers/2017-02-28-single-shot-detector-ssd.md

```diff
@@ -0,0 +1,47 @@
+
+
+```
+---
+layout: review
+title:  SSD: Single Shot MultiBox Detector (2016)
+author: Frédéric Branchaud-Charron
+link:   https://arxiv.org/pdf/1512.02325.pdf
+tags:   convolutional-networks deep-learning object-detection bounding-boxes
+---
+```
+
+Single shot MultiBox Detector
+
+## Summary
+
+**SSD** is an adaptation of **YOLO** to support prior shapes. Using prior shapes, **SSD** doesn't need to learn the aspect ratio, it predicts the correct prior box and the class. Also, SSD is using multiples features map to achieve a better performance.
+
+## Model
+
+The model is using the **VGG-16** model for its base. It then uses severals features maps to produce its output.
+
+It's a more complex model than **YOLO** but it's faster because the input size is smaller.
+
+![ssd_model][/deep-learning/images/ssd_model.png]
```

Comment:
Surround the url with `()`:
`![ssd_model](/deep-learning/images/ssd_model.png)`